awx/settings/defaults.py

```diff
@@ -459,40 +457,9 @@ def IS_TESTING(argv=None):
 # Set default ports for live server tests.
 os.environ.setdefault('DJANGO_LIVE_TEST_SERVER_ADDRESS', 'localhost:9013-9199')
 
-djcelery.setup_loader()
-
 BROKER_POOL_LIMIT = None
 BROKER_URL = 'amqp://guest:guest@localhost:5672//'
-CELERY_EVENT_QUEUE_TTL = 5
 CELERY_DEFAULT_QUEUE = 'awx_private_queue'
```

Comment:
I kept a few of these setting names w/ `CELERY_` because it was just simpler, and I didn't want to muck with setting names, given their historic fragility.